data/battle_scripts_1.s

```diff
@@ -466,6 +467,62 @@ BattleScript_SyrupBombTurnDmgEnd:
 BattleScript_EffectMatchaGotcha::
 	setmoveeffect MOVE_EFFECT_BURN
 	goto BattleScript_EffectAbsorb
+  end
+
+BattleScript_EffectChillyReception::
+	printstring STRINGID_PKMNTELLCHILLINGRECEPTIONJOKE
+	attackcanceler
+	waitmessage B_WAIT_TIME_LONG
+	attackstring
+	ppreduce
+	jumpifhalfword CMP_COMMON_BITS, gBattleWeather, B_WEATHER_SUN_PRIMAL, BattleScript_ExtremelyHarshSunlightWasNotLessenedRet
+	jumpifhalfword CMP_COMMON_BITS, gBattleWeather, B_WEATHER_RAIN_PRIMAL, BattleScript_EffectChillyReceptionBlockedByPrimalRain
+	jumpifhalfword CMP_COMMON_BITS, gBattleWeather, B_WEATHER_STRONG_WINDS, BattleScript_MysteriousAirCurrentBlowsOnRet
+	attackanimation
+	waitanimation
+	setsnow
+	call BattleScript_MoveWeatherChangeRet
+	goto BattleScript_MoveSwitch
+BattleScript_EffectChillyReceptionBlockedByPrimalSun:
+  call BattleScript_ExtremelyHarshSunlightWasNotLessenedRet
+  goto BattleScript_MoveSwitch
```

Comment:
If you jump directly the switchout, you're skipping the attack animation

Reply:
That was actually by design because other weather skip their animation when they fail. I did update it to always play the Chilly Reception animation because of this feedback and the switching portion of the move "always" happening